backup_and_restore/index.adoc

```diff
@@ -27,3 +26,32 @@ You might run into several situations where {product-title}  does not work as ex
 * You have lost the majority of your control plane hosts, leading to etcd quorum loss.
 
 You can always recover from a disaster situation by xref:../backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.adoc#dr-restoring-cluster-state[restoring your cluster to its previous state] using the saved etcd snapshots.
+
+[id="application-backup-restore-operations-overview"]
+== Application backup and restore operations
+
+As a cluster administrator, you can back up and restore applications running on {product-title} by using the OpenShift API for Data Protection (OADP).
+
+OADP backs up and restores Kubernetes resources, internal images, and cluster metadata, at the granularity of a namespace, by using a supported version of link:https://velero.io/[Velero]. See xref:../backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc#oadp-features_oadp-features-plugins[OADP features].
+
+OADP backs up and restores PVs by using the cloud provider snapshot API. You can install OADP with xref:../backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc#oadp-plugins_oadp-features-plugins[default plugins].
+
+.OADP requirements
+
+OADP has the following requirements:
+
+* You must be logged in as a user with a `cluster-admin` role.
+* You must have a supported object storage for storing backups:
+
+** Amazon Web Services
+** Microsoft Azure
+** Google Cloud Platform
+** Multicloud Object Gateway, a component of OpenShift Container Storage
+** S3-compatible object storage, for example, Noobaa or Minio
+
+* You must have a cloud provider that supports snapshots:
```

Comment:
Unsure if we want this to say "you must". Technically a user can default to restic for filesystem backup/restore.

Reply:
I was going to change this to "cloud provider that supports snapshots or the Container Snapshot Interface".

Re: Restic. I have not mentioned it because I have not heard that we are documenting it for GA. Can you confirm one way or the other?